### ISSUE TITLE
I cannot for the life of me understand these matches

### ISSUE THREAD

**Issue by michardy** [michardy/sources-bot#3]
Trump has 'inside information' on who protested in Charlottesville, says Falwell Jr. in president's defense
Other sources for this story:

1. [Kasich downplays notion of 2020 challenge to Trump, saying president can learn from challenges](https://www.washingtonpost.com/news/post-politics/wp/2017/08/20/kasich-downplays-notion-of-2020-challenge-to-trump-saying-president-can-learn-from-challenges/)
2. [Mark Lilla says an obsession with identity politics has wrecked US liberalism](https://www.theguardian.com/society/2017/aug/20/once-and-future-liberal-reviews-identity-politics-mark-lilla)

**Comment by michardy** [comment]
**Trump has** 'inside information' on who protested in Charlottesville, **says** Falwell Jr. in president's defense
Other sources for this story:

1. [Kasich downplays notion of 2020 challenge to **Trump**, saying president can learn from challenges](https://www.washingtonpost.com/news/post-politics/wp/2017/08/20/kasich-downplays-notion-of-2020-challenge-to-trump-saying-president-can-learn-from-challenges/) (potentially **president**'s matches with president, I have to check the tokenizer)
2. [Mark Lilla **says** an obsession with identity politics **has** wrecked US liberalism](https://www.theguardian.com/society/2017/aug/20/once-and-future-liberal-reviews-identity-politics-mark-lilla) (Two matching verbs)

**Comment by michardy** [comment]
Yep. president's becomes president 's
``` python
['Trump', 'has', "'inside", 'information', "'", 'on', 'who', 'protested', 'in', 'Charlottesville', ',', 'says', 'Falwell', 'Jr.', 'in', 'president', "'s", 'defense']
```

**Comment by michardy** [comment]
Need to remove things like has and is from entities list.  
Also president and Trump should be de-duplication targets.  In other words `Trump` or the current president should be replaced with `president` so that more than one reference to it is removed

**Comment by michardy** [comment]
Fixed by 303c4b382fdcd2584eee231e2680f790bbc8fb38
